packages/lambda-xyz/src/path.ts

```diff
@@ -0,0 +1,20 @@
+import * as Path from 'path-to-regexp';
+
+const XyzPath = Path('/:group/:z/:x/:y\\.:ext');
```

Comment:
Sorry for my late drive by comment.. but have we got a design of public target API? Would be good to confirm what group is? Also it's possible we will support multiple projections in the future and maybe the path should design for this? I only raise this as the public API is very important and you don't want to be changing it too much. 

For some sort of example, LDS uses this schema for tile services: https://help.koordinates.com/query-api-and-web-services/using-koordinates-xyz-and-wmts-tiles-services-javascript-clients/. Not saying this is what we should use, but a good reference

Reply:
I am happy to change the URL,  It is not designed to be public, was just trying to get something going.

In the future we will likely need a large number of options like projection & scale so using something like the coordinates system would be a good idea.

Once we learn more about our requirements we can change the url to be better suited, this is an internal testing api :)

Reply:
Just wanted to raise it now. Cool if it's internal only